### ISSUE TITLE
Engine Request - NBlood

### ISSUE THREAD

**Issue by neuromancer** [dreamer/luxtorpeda#20]
Engine: https://github.com/nukeykt/NBlood

Game: https://store.steampowered.com/app/1010750/Blood_Fresh_Supply/

**Comment by dreamer** [comment]
Isn't this game going to have native Linux release soon? In my experience, Nightdive studios has an excellent record in supporting Linux games.

**Comment by neuromancer** [comment]
Yes, soon™, but in IMHO, NBlood is better than the Nightdive re-implementation (which I don't know if it was based on the original code or reimplemented using Eduke32 or from scratch..)

**Comment by neuromancer** [comment]
Btw, I found that there is no indication NDS will continue to update the game (https://steamcommunity.com/games/1010750/announcements/detail/1611645072188616319)

> Thank you for your patience while we were dealing with the remaining issues. For future issues, we will have to recommend contacting Atari about them.

They clarified this in their [discord](https://i.imgur.com/Eexiyzi.jpg) channel.

**Comment by dreamer** [comment]
So basically Atari pulled the plug? :(

**Comment by neuromancer** [comment]
It's unclear to me, but maybe checking this [long post](https://steamcommunity.com/app/1010750/discussions/0/1640918469767087271/) has some answers.

**Comment by neuromancer** [comment]
https://www.reddit.com/r/linux_gaming/comments/d7qgyq/blood_fresh_supply_returned_to_atari_then_half/

**Comment by dreamer** [comment]
Thanks for pointing this out. Sad news :(

ATM I am working on dosbox-related issues, I will try packaging NBlood in a few days (if someone else wants to give it a go and does not know where to start, then ping me).
